docker-compose/compose.yaml

```diff
@@ -84,8 +80,15 @@ services:
         http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file
     ports:
       - "8095:8095"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8095/actuator/health"]
+      interval: 60s
```

Comment:
I would avoid more than 30 seconds, it means you risk losing up to 59s for the next check to return healthy

I would try to stick to 10 seconds checks for simpler services and have 30 seconds only for the slowest to start